### ISSUE TITLE
ASUS Crosshair VI Hero - missing Fan channel 5 (fan 6)

### ISSUE THREAD

**Issue by spec-chum** [LibreHardwareMonitor/LibreHardwareMonitor#710]
I've got my D5 pump on there, and it shows up in Argus but I can't get it to show in LHM.

![image](https://user-images.githubusercontent.com/4376135/164965923-d4e56873-b54d-49ae-abaf-5eb116915a6d.png)

**Comment by quangkieu** [comment]
Yeah I think it is Asus CPU_OPT problem. My x570 tuf show the fan but I have no control over
